cmd/uid_provider/api/controllers/routes.go

```diff
@@ -0,0 +1,18 @@
+package controller
+
+import handler "gitlab.inspr.dev/inspr/core/cmd/uid_provider/api/handlers"
+
+// initRoutes defines which routes the UID Provider API will have
+func (s *Server) initRoutes() {
+	h := handler.NewHandler(s.Rdb)
+
+	s.Mux.HandleFunc("/newuser", h.CreateUserHandler)
+
+	s.Mux.HandleFunc("/deleteuser", h.DeleteUserHandler)
+
+	s.Mux.HandleFunc("/updatepwd", h.UpdatePasswordHandler)
+
+	s.Mux.HandleFunc("/login", h.LoginHandler)
+
+	s.Mux.HandleFunc("/refreshtoken", h.RefreshTokenHandler)
```

Comment:
This does not assure of the method that is being called, wrap it on the correct method.

You can change the handlers to be of the `rest.Handler` type, and just call h.Method.POST() or something like it.

Reply:
Yeah, was talking about this with @LucasStarlingdePaulaSalles yesterday, will use the same handler structure used in Insprd